doc/tutorial/fcm_fundamentals

```diff
@@ -0,0 +1,859 @@
+                written for users moving from CVS to Subversion (SVN).
+                <br/>
+                <br/>
+                <h4>Summary of svn:</h4>
+                <li>It is an incremental change tracking system.</li>
+                <li>We can explore the changes at each change and recall a version using its 
+                    revision number.
+                </li>
+                <li>It uses a repository to store the files, kind of like a file server and 
+                    is known as a the filesystem tree.
+                </li>
+                <li>A working copy, which is in fact, a local copy of a particular version of 
+                    the repository is availiable for users to freely work on.
+                    Note managing the version control of the working copy and communicating 
+                    changes is the responsibility of the user.
+                </li>
+                <li>Using copy-modify-merge allows each user of the repository to have their 
+                    own personal working copy and they can work simultaneously and independently 
+                    to modify their personal copies which can then be merged together later. 
+                    Note it is the humans reponsibility to merge the files correctly and 
+                    resolve any conflicts by manually choosing the correct solution after 
+                    usually after discussions with the other people editing involved.
+                </li>
+                <li>Each repository change which is committed to the repository creates a new 
+                    state of the filesystem tree, this is called a revision number.
+                </li>
+                <li>The revision number is applied to the entire repository tree and not 
+                    individual files and is therefore also known as the Global Revision Number.
+                    For example: at revision number N then a file may be the same as it was at 
+                    revision number M but a different file was changed between the two 
+                    revisions.
+                </li>
+                <br/>
+                Version Control information
+                <br/>
+                The <a href="http://svnbook.red-bean.com/">Red bean book</a> describes a 
+                version control system.
+                <br/>
+            </div>
+            </p>
+        </ul>
+        <h1><a id="S2">The basics of svn and where to learn more</a></h1>
+        <p>
+        <ul>
+            <div class="page-header">
+                <h3>Files</h3>
+            </div>
+            <div class="well">
+                <li>File sizes</li>
+                Large files and file that are constantly growing can be an issue. 
+                <br/>
+                Files greater than 10MB are not allowed to be committed to the repository 
```

Comment:
@matthewrmshin - is the 10MB something site configurable? If so I'm not sure we can say this.